<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>711</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>707</width>
          <height>507</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarHorizontal">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionlabel"/>
   <addaction name="separator"/>
   <addaction name="actionFile"/>
   <addaction name="actionFolder"/>
   <addaction name="actionGt"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBarVertical">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionlabel_2"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>file</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="text">
    <string>folder</string>
   </property>
  </action>
  <action name="actionGt">
   <property name="text">
    <string>loading</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionlabel">
   <property name="text">
    <string>AutoLabel</string>
   </property>
  </action>
  <action name="actionlabel_2">
   <property name="text">
    <string>AutoLabel</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionAdjust">
   <property name="text">
    <string>adjust size</string>
   </property>
  </action>
  <action name="actionAnno">
   <property name="text">
    <string>annotation</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>model</string>
   </property>
  </action>
  <action name="actionDetection">
   <property name="text">
    <string>detection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
